packages/lib/gpcircuits/test/common.ts

```diff
@@ -17,10 +18,16 @@ export const privateKey = "AAECAwQFBgcICQABAgMEBQYHCAkAAQIDBAUGBwgJAAE"; // hex
 
 export const privateKey2 = "AAEBAQIDBQEGABABAgMEBQYHCAkAAQIDBAUGBAgJAAA"; // hex 0001010102030501060010010203040506070809000102030405060408090000
 
+// Semaphore V3 identity
 export const ownerIdentity = new Identity(
   '["329061722381819402313027227353491409557029289040211387019699013780657641967", "99353161014976810914716773124042455250852206298527174581112949561812190422"]'
 );
 
+// Semaphore V4 identity
+export const ownerIdentityV4 = new IdentityV4(
+  "0001020304050607080900010203040506070809000102030405060708090001"
```

Comment:
I think this +1 trick might be used for a POD private key somewhere else.  Pick a different input pattern just to be sure we never accidentally end up with the same key where they could be different.  I worry that could cause confusion in tests.